_sass/tiendatek-theme.scss

```diff
@@ -479,6 +479,7 @@ footer {
 // About
 .intro {
   font-size: 1.5em;
+  text-align: justify;
```

Comment:
Many experts say that we should never use `justify` in web, since browsers can't split words in the middle, and sometimes there are unwanted side effects because of that.

Reply:
I didn't know it, but I read some articles and you are right. Thanks you!